notebooks/pytorch_sparse_quantized_transfer_learning.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<p>Let's remove the mention of trying out their model. We want to highlight transfer learning only here, so can assume the model must come from the SparseZoo and the user can swap it out for their own dataset</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuralmagic/sparseml/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>